package.json

```diff
@@ -16,8 +16,10 @@
     "envc": "2.5.0",
     "escape-html": "1.0.3",
     "express": "4.15.2",
+    "habitat": "^3.1.2",
```

Comment:
We've generally been using exact versions, then upgrading in batch, so that we know everyone has the exact same version.